creusot-contracts/src/logic/mapping.rs

```diff
@@ -24,30 +24,53 @@ impl<A: ?Sized, B: ?Sized> Mapping<A, B> {
     #[trusted]
     #[logic]
     #[creusot::builtins = "map.Map.get"]
+    #[allow(unused_variables)]
     pub fn get(self, a: A) -> B
     where
         B: Sized, // TODO : don't require this (problem: return type needs to be sized)
     {
-        let _ = a;
         dead
     }
 
     /// Returns a new mapping, where `a` is now mapped to `b`.
     #[trusted]
     #[logic]
     #[creusot::builtins = "map.Map.set"]
+    #[allow(unused_variables)]
     pub fn set(self, a: A, b: B) -> Self {
-        let _ = a;
-        let _ = b;
         dead
     }
 
     /// Create a mapping, where every value of type `a` is mapped to `b`.
     #[trusted]
     #[logic]
     #[creusot::builtins = "map.Const.const"]
+    #[allow(unused_variables)]
     pub fn cst(b: B) -> Self {
-        let _ = b;
         dead
     }
 }
+
+impl<T> Mapping<T, bool> {
+    /// Creates a logical value satisfying the property given by `self`.
+    ///
+    /// This is also called the "epsilon operator": its goal is not extract from `∃x. P(x)`
+    /// a `x` satisfying `P`.
+    ///
+    /// This is the method version of [`such_that`](crate::util::such_that).
+    #[logic]
+    #[open]
+    pub fn such_that(self) -> T {
+        crate::util::such_that(self)
+    }
+}
```

Comment:
Quite frankly, I am not convinced there is a need for this. This is redundant, and it does not allow to write shorter code.